### ISSUE TITLE
[Feature] Ability to see Recent Moves in the Budget

### ISSUE THREAD

**Issue by atgrey24** [actualbudget/actual#3043]
### Verified feature request does not already exist?

- [X] I have searched and found no existing issue

### 💻

- [ ] Would you like to implement this feature?

### Pitch: what problem are you trying to solve?

As a User, I would like to see recent budget moves such as money assigned to a category or moved between categories, so that I can better understand the current state of my budget or potentially revert previous moves.

The undo function is useful for correcting an immediate step, but is not helpful for reviewing all moves that were made. It also does not work across sessions or devices.

### Describe your ideal solution to this problem

Have a popup/modal that displays all recent budget actions for the past 30 days, with the ability to filter between "Budgeted" (money moved to/from To Be Budgeted) or "Moved" (money moved between categories), or view all. This could be another item in the month's drop down menu, or it's own button/toggle similar to notes. Name of the feature could be "Recent Moves", "Move History", "Budget Activity" or something similar.

### Teaching and learning

This type of feature should be fairly intuitive if named properly, though references to it should be added to the documentation.

**Comment by joel-jeremy** [comment]
A budget activity/history would be a very good and useful feature to have. I could see this log be used in some sort of a report to visualize how users move money throughout the month. 

One simple implementation that comes to mind is to leverage the notes of the categories or introduce a new note for the Budgeted column that resets every month. Every movement to/from the Budgeted amount will be logged on the notes automatically.

On the other hand, having an activity table would be a more powerful option. It would enable the app to sort of reverse the movements of the budgets and will be easier to use in a report.

**Comment by nakermann1973** [comment]
This is the same as #2602 - The way I see it is this is asking for a custom notes field per month for each category in the monthly view, with the following possible behavior:

* Templates could auto-populate this note when initializing the budget category for a month.  
* "Copy" or "set budgets" actions could auto-populate the note
* "Transfer to another category" could record this in the note
* Manual entries to the note would be recorded.

I have upvoted both feature requests

**Comment by alex-t-grey** [comment]
@nakermann1973 its turns out that "Transfer to another category" moves are already recorded in the month's note. Though it only records moves when clicking that action (or the "cover overspending" action which is really the same thing). I don't know if that was implemented before or after I made this feature request.

I don't really follow why a second note section per category would be necessary. I also disagree that this is the same feature request as the one you linked. I guess it could be possible to fulfil both requests with a single solution, but they're two different user stories.
